helm/alfresco-content-services/values.yaml

```diff
@@ -662,7 +628,7 @@ dtas:
         base_path: "/workspace"
       aas:
         audit_host: http://acs-alfresco-audit-storage:8081
-        elasticsearch_host: http://elasticsearch-aas-master:9200
+        elasticsearch_host: http://acs-elasticsearch:9200
```

Comment:
this will fail in case the release name is not "acs". We should probably turn dtas.config into a string and use a tpl() function to render the dtas config json